tests/rt_utils.sh

```diff
@@ -442,49 +444,52 @@ rocoto_create_compile_task() {
   if [[ "Q$APP" != Q ]] ; then
       rocoto_cmd="&PATHRT;/appbuild.sh &PATHTR;/FV3 $APP $COMPILE_NR"
   else
-      rocoto_cmd="&PATHRT;/compile.sh &PATHTR;/FV3 $MACHINE_ID \"${NEMS_VER}\" $COMPILE_NR"
+      rocoto_cmd="&PATHRT;/compile_cmake.sh &PATHTR; $MACHINE_ID \"${NEMS_VER}\" $COMPILE_NR"
   fi
 
   NATIVE=""
+  BUILD_CORES=8
   if [[ ${MACHINE_ID} == wcoss_dell_p3 ]]; then
-    NATIVE="<memory>6G</memory> <native>-R 'affinity[core(1)]'</native>"
+    BUILD_CORES=1
+    NATIVE="<memory>8G</memory> <native>-R 'affinity[core(1)]'</native>"
+  fi
+  if [[ ${MACHINE_ID} == wcoss_cray ]]; then
+    BUILD_CORES=24
+    rocoto_cmd="aprun -n 1 -j 1 -N 1 -d $BUILD_CORES $rocoto_cmd"
+    NATIVE="<exclusive></exclusive>"
+  fi
+  BUILD_WALLTIME="00:30:00"
+  if [[ ${MACHINE_ID} == jet ]]; then
+    BUILD_WALLTIME="01:00:00"
   fi
 
-  if [[ ${COMPILE_NR_DEP} -gt 0 ]]; then
-    cat << EOF >> $ROCOTO_XML
-  <task name="compile_${COMPILE_NR}" maxtries="1">
-    <dependency> <taskdep task="compile_${COMPILE_NR_DEP}"/> </dependency>
-    <command>$rocoto_cmd</command>
-    <jobname>compile_${COMPILE_NR}</jobname>
-    <account>${ACCNR}</account>
-    <queue>${COMPILE_QUEUE}</queue>
-    <cores>1</cores>
-    <walltime>01:00:00</walltime>
-    <join>&LOG;/compile_${COMPILE_NR}.log</join>
-    ${NATIVE}
-  </task>
-EOF
-  else
-    cat << EOF >> $ROCOTO_XML
-  <task name="compile_${COMPILE_NR}" maxtries="1">
+  # serialize CCPP builds. FIXME
```

Comment:
This should be ok in parallel now that we have the code in ccpp-physics and ccpp-framework. I can try to add the missing bits after this PR has been merged.

Reply:
Removed outdated comments about serialized ccpp build jobs. See e7dcda0